pkg/crc/preflight/preflight.go

```diff
@@ -56,60 +54,60 @@ func preflightCheckAndFix(check, fix PreflightCheckFixFuncType, message string)
 
 // StartPreflightChecks performs the preflight checks before starting the cluster
 func StartPreflightChecks() {
-	preflightCheckSucceedsOrFails("",
+	preflightCheckSucceedsOrFails(config.GetViperBool(cmdConfig.SkipCheckVirtEnabled.Name),
 		checkVirtualizationEnabled,
 		"Checking if Virtualization is enabled",
-		"",
+		config.GetViperBool(cmdConfig.WarnCheckVirtEnabled.Name),
```

Comment:
we might want to consider this to change down the line...
we still use config that is typed by naming... and located in `cmd`.
for now this is better than before...